tests/test_repo.py

```diff
@@ -116,8 +110,6 @@ def test_repo_add_success(qapp, qtbot, mocker, borg_json_output, keyring_fixture
     keyring = VortaKeyring.get_keyring()
     assert keyring.get_password("vorta-repo", RepoModel.get(id=2).url) == LONG_PASSWORD
```

Comment:
gives me a assertion error here.

Reply:
In that case you should debug the test and see *why* it fails. Looking at the compared values would be a first good step.